docker/Dockerfile.windows

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:16.04
```

Comment:
What is that file used for?
The windows build is already done in the existing script.
Please don't rewrite thing that works your way. it's not helpful at all.

Reply:
That's exactly the cause for this ;) Sure, not a problem.
The idea was to eventually create two pre-defined Docker images with all the needed libs and tools pre-installed (two images = lower size = faster download = quicker build times/effects). Dropping the idea.